python/pyspark/sql/dataframe.py

```diff
@@ -1422,6 +1422,61 @@ def colRegex(self, colName: str) -> Column:
         jc = self._jdf.colRegex(colName)
         return Column(jc)
 
+    def to(self, schema: StructType) -> "DataFrame":
+        """
+        Returns a new :class:`DataFrame` where each row is reconciled to match the specified
+        schema.
+
+        Spark will:
```

Comment:
Can we switch this to:

```
Note
----
```

? Let's also remove `Spark will:`

Reply:
got it.